wordpress-mobile/WordPress-iOS
# Integrate media picker 0.25

Update to version 0.25 of the picker. Details of changes here: https://github.com/wordpress-mobile/MediaPicker-iOS/releases/tag/0.25

To test:
 - Check if the main uses of the picker are all correct:
 - Media Library
 - Site icon picker
 - Gravatar user picker
 - Edit Post, media picker
